recipe/meta.yaml

```diff
@@ -1,40 +1,57 @@
-{% set name = "Metaphone" %}
+{% set name = "metaphone" %}
```

Comment:
You can technically keep it uppercase because the name is defined with `name: {{ name|lower }}`.

Reply:
if it's not mandatory I'd rather have the name as it should be

Reply:
As usual, I will recommend/ask to keep the diff as small as possible with conda-forge.

Reply:
agree, I restored it as in conda-forge, thank you for the context